migrations/overview.md

```diff
@@ -0,0 +1,147 @@
+# Set Protocol Contract Deployment
+
+## Token White List
+
+For regulatory and technical reasons, only certain tokens are allowed to be used in the protocol. WETH and WBTC are the initial tokens.
+
+Script should:
+1. Get addresses for WBTC and WETH based on current network.
+2. Deploy "WhiteList" contract with array containing addresses of approved tokens
+
+## Core
+
+Before deploying contracts, a few libraries are needed. They include:
+  - `ERC20Wrapper` is used extensively to deal with tokens
+  - `EIP712Library` for typed structure data hashing and signing
+  - `OrderLibrary` for validating and hashing issuance orders (when someone has the required components in the vault and finally wants to create a set)
+  - `ExchangeIssueLibrary` helps with parsing exchange issuance data. Only contains a struct atm.
+  - `StandardProposeLibrary` contains all the logic when a set needs to be rebalanced. Proposal is the first stage and is required to be called by an outside. Restrictions such as whether a manager is proposing, the rebalance is at the correct time etc are all checked here.
+  - `StandardStartRebalanceLibrary` once a certain time period has elapsed from the time of proposal. Rebalancing begins, issuance/withdrawals are disabled and the the dutch auction is setup. 
+  - `StandardPlaceBidLibrary` allows you to place a bid in the dutch auction
+  - `StandardSettleRebalanceLibrary` once the auction is finished, the set now needs to be settled.
+  - `StandardFailAuctionLibrary` in case the auction fails because no one can place bids (paused tokens) the protocol will return a Set to it's default state or put it in "drawdown" which allows the underlying collateral to be redeemed. 
+  - `RebalancingHelperLibrary` is used by all the rebalancing libraries as a helper.
+
+The script for core contract deployments should:
+  1. Deploy the ERC20Wrapper then link it to all the other contracts that require it
+  2. Deploy the EIP712Library then link it to all the other contracts that require it
+  3. Deploy the OrderLibrary then link it to all the other contracts that require it
+  4. Deploy the ExchangeIssueLibrary then link it to all the other contracts that require it
+  5. Deploy each Rebalancing Library then link it to the Rebalancing Helper library
+  6. Link each Rebalancing Library (including helper) to the RebalancingSetTokenFactory
+  7. Link each Rebalancing Library (including helper) to the RebalancingSetToken
+  8. Deploy a SignatureValidator contract which decodes a signature from a hash to make sure it matches with a supplied address
```

Comment:
Update README to remove the OrderLibrary, SignatureValidator, EIP712Library, etc.